packages/pf-codemods/README.md

```diff
@@ -740,6 +740,28 @@ Out:
 <FileUpload  />
 ```
 
+### formgroup-remove-helpertextProps [(#8810)](https://github.com/patternfly/patternfly-react/pull/8810)
+
+We've removed the helper text related props from `FormGroup`. This rule will remove the `helperText`, `helperTextInvalid`, `validated`, `helperTextIcon`, `helperTextInvalidIcon`, and `isHelperTextBeforeField` props if present. The `FormHelperText`, `HelperText`, and `HelperTextItem` components should now be used directly as part of `children` instead of these props.
```

Comment:
This just needs to be updated since we're not removing them anymore. Should also add, "This rule will throw an error, but will not make any changes." at the end.